packages/augur-ui/src/services/augurjs.js

```diff
@@ -1,25 +1,25 @@
 import Augur from "@augurproject/augur.js";
```

Comment:
should prob convert this file to `augurjs.ts`

Reply:
converted to .ts file